coordinator/src/position/models.rs

```diff
@@ -49,4 +56,198 @@ pub struct Position {
     /// This field is optional for backwards compatibility because we cannot deterministically
     /// associate already existing contracts with positions.
     pub temporary_contract_id: Option<ContractId>,
+    pub closing_price: Option<f32>,
+}
+
+impl Position {
+    /// Calculates the profit and loss for the coordinator
```

Comment:
It's my understanding that this function returns the PnL in satoshi. 

```suggestion
    /// Calculates the profit and loss for the coordinator in satoshi
```